public/app/features/profile/UserProfileEditPage.tsx

```diff
@@ -66,6 +78,9 @@ export function UserProfileEditPage({
           <UserTeams isLoading={teamsAreLoading} teams={teams} />
           <UserOrganizations isLoading={orgsAreLoading} setUserOrg={changeUserOrg} orgs={orgs} user={user} />
           <UserSessions isLoading={sessionsAreLoading} revokeUserSession={revokeUserSession} sessions={sessions} />
+          {extensionComponents.map(({ component: Component }, index) => (
+            <Component key={index} />
```

Comment:
I wonder if we could pass `key` as a parameter, optionally, so that if the extension wants to force a re-render here it can. other than that, this is great!